### ISSUE TITLE
Notices behind the scenes

### ISSUE THREAD

**Issue by parljohn** [getminera/minera#217]
Just did a clean install on raspberry pi and I'm trying to add network servers or scan for them.  Nothing comes back and leaves a waiting screen.  Behind the scenes I see this output (how do I get rid of notices or do you just need to fix the code?) :

<div style="border:1px solid #990000;padding-left:20px;margin:0 0 10px 0;">
<h4>A PHP Error was encountered</h4>
<div style="border:1px solid #990000;padding-left:20px;margin:0 0 10px 0;">
<h4>A PHP Error was encountered</h4>

<p>Severity: Notice</p>
<p>Message:  Undefined offset: 6</p>
<p>Filename: controllers/app.php</p>
<p>Line Number: 407</p>
</div>

{"minerd_software":"cpuminer","network_miners":"[{\"name\":\"1\",\"ip\":\"192.168.0.134\",\"port\":\"\",\"algo\":\"SHA-256\",\"type\":\"newAnt\",\"pools\":[]},{\"name\":\"2\",\"ip\":\"192.168.0.135\",\"port\":\"\",\"algo\":\"SHA-256\",\"type\":\"newAnt\",\"pools\":[]},{\"name\":\"3\",\"ip\":\"192.168.0.133\",\"port\":\"\",\"algo\":\"SHA-256\",\"type\":\"newAnt\",\"pools\":[]},{\"name\":\"4\",\"ip\":\"192.168.0.139\",\"port\":\"\",\"algo\":\"SHA-256\",\"type\":\"newAnt\",\"pools\":[]},{\"name\":\"5\",\"ip\":\"192.168.0.140\",\"port\":\"\",\"algo\":\"SHA-256\",\"type\":\"newAnt\",\"pools\":[]},{\"name\":\"6\",\"ip\":\"192.168.0.114\",\"port\":\"\",\"algo\":\"Scrypt\",\"type\":null,\"pools\":[]}]","custom_miners":false,"manual_options":"0","guided_options":"1","minerd_log":false,"minerd_autodetect":"1","minerd_autotune":false,"minerd_startfreq":"800","minerd_extraoptions":"","pool_global_proxy":"","minerd_json_settings":"{\"gc3355-detect\":true,\"freq\":\"800\",\"pools\":[{\"url\":\"stratum+tcp:\\\/\\\/us.multipool.us:7777\",\"user\":\"michelem.minera\",\"pass\":\"x\"}]}","minerd_autorecover":false,"minerd_use_root":false,"minerd_autorestart":false,"minerd_autorestart_devices":"","minerd_autorestart_time":"600","minera_donation_time":"10","dashboard_refresh_time":"60","dashboard_temp":"c","dashboard_skin":"black","dashboard_table_records":"5","dashboard_devicetree":"1","dashboard_box_profit":"1","dashboard_box_local_miner":false,"dashboard_box_local_pools":false,"dashboard_box_network_details":"1","dashboard_box_network_pools_details":"1","dashboard_box_chart_shares":"1","dashboard_box_chart_system_load":"1","dashboard_box_chart_hashrates":"1","dashboard_box_scrypt_earnings":"1","dashboard_box_log":"1","minerd_pools":"[{\"url\":\"stratum+tcp:\\\/\\\/us.multipool.us:7777\",\"username\":\"michelem.minera\",\"password\":\"x\"}]","minera_timezone":"Africa\/Abidjan","minerd_delaytime":5,"system_extracommands":false,"scheduled_event_time":false,"scheduled_event_action":false,"anonymous_stats":"1","mobileminer_enabled":false,"mobileminer_system_name":false,"mobileminer_email":false,"mobileminer_appkey":false,"minera_id":"nnjmmniogywm"}

**Comment by michelem09** [comment]
Yes I know this bug, but unfortunately I can't do anything now and for the next months (out of home).
You have to manually add your miners. Sorry for that.

**Comment by parljohn** [comment]
Unfortunately I can't even add manually now.  It just keeps throwing the same notice and now it duplicates my miner entries and removes old ones.  I may do a pull request and try fixing it myself, let me know if this is something you can fix.

**Comment by parljohn** [comment]
I would be happy to donate some BTC if I could actually just get it to work.

**Comment by michelem09** [comment]
Please tell me what version of Minera you are using and how you installed it (by official image or manually?).
Then kind of miners and if they are USB ones what kind of miner software you are trying.
